api/api.md

```diff
@@ -879,6 +890,7 @@ namespace OpenAI.Chat {
         public ChatFunctionCall FunctionCall { get; init; }
         public string ParticipantName { get; init; }
         public IList<ChatToolCall> ToolCalls { get; }
+        protected override void WriteCore(Utf8JsonWriter writer, ModelReaderWriterOptions options);
```

Comment:
This file needs to be regenerated before merging.

Reply:
Regenerated, hand-copying AssistantClient and VectorStoreClient. Note that this still looks a little "dirty" because it's not fresh in main -- this WriteCore [does exist in code](https://github.com/openai/openai-dotnet/blob/59c8433084d9bdb4bac62a48ab39942be0c54060/src/Custom/Assistants/ToolDefinition.Serialization.cs#L18) despite [not currently being in the .md](https://github.com/openai/openai-dotnet/blob/59c8433084d9bdb4bac62a48ab39942be0c54060/api/api.md?plain=1#L638).